### PR TITLE
Added FAQ for mac error message when installing

### PR DESCRIPTION
**Pull request by AaronCheung430**
Hi there,

I have added this question to the FAQ in case someone else needed help with it when installing.

Thank you

### CODE REVIEW

**Review by nathanaelhoun — APPROVED** [review]
LGTM, maybe a mac user (@vraravam?) can confirm this solution?

File: pages/faq.tsx [hunk 1]
```diff
@@ -105,6 +105,17 @@ function FAQ({}: Props) {
             guide.
           </>
         </FAQEntry>
+        <FAQEntry title='I received a "Ferdium is damaged and can’t be opened. You should move it to the Trash” error message when trying to install Ferdium. What should I do?'>
+          <>
+            This error message is shown because the releases aren't yet signed. We are still working on getting an Apple developer license.
+            <br />
+            Meanwhile, you could check out{" "}
+            <ExternalLink href='https://github.com/ferdium/ferdium-app/issues/34'>
```
Comment (vraravam) [inline pages/faq.tsx:113]: There's an entry in the main `README.md` here for this https://github.com/ferdium/ferdium-app#unsigned-nightlies-on-macos - I prefer pointing to the README as opposed to an issue for such FAQ

Comment (AaronCheung430) [reply pages/faq.tsx:113]: Thank you and it's a great idea, I didn't notice there is a section about it in the main `README.md`. I will reword this faq to be more general for both Windows and macOS, and link it to that section in the `README.md`, if that sounds good?

Comment (vraravam) [reply pages/faq.tsx:113]: yes - please go ahead! Thanks!

**Review by AaronCheung430 — COMMENTED** [review]
Just updated with commit. Hi guys, sorry, I am new to open source. I am not sure whether I am doing things right, as I am trying out features. Am I supposed to submit another review, like this one after I committed? or what will usually be the case? Do I need to resolve those conversation above as well? Thank you

**Comment by vraravam** [comment]
> Just updated with commit. Hi guys, sorry, I am new to open source. I am not sure whether I am doing things right, as I am trying out features. Am I supposed to submit another review, like this one after I committed? or a comment? or what will usually be the case? Do I need to resolve those conversation above as well? Thank you

Once anyone interacts with an open PR, they keep getting notified for new updates (like how I got an email notification). So, maintainers will usually come back (uppon notification) to re-review the new changes. You are all good from that point of view.

**Review by vraravam — APPROVED** [review]

File: pages/faq.tsx [hunk 2]
```diff
@@ -105,6 +105,17 @@ function FAQ({}: Props) {
             guide.
           </>
         </FAQEntry>
+        <FAQEntry title='I received a "Ferdium is damaged and can’t be opened. You should move it to the Trash” error message when trying to install Ferdium. What should I do?'>
```
[the remark this answers is not in the document]
Comment (vraravam) [reply pages/faq.tsx]: I would not want to introduce Microsoft's weird double quotes - can this be reverted please?

[the remark this answers is not in the document]
Comment (vraravam) [reply pages/faq.tsx]: Looks like this was from an older version - the latest commit uses the old (ASCII-style) double quotes. Thanks!

**Comment by vraravam** [comment]
@all-contributors please add @AaronCheung430 for documentation

File: pages/faq.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (AaronCheung430) [reply pages/faq.tsx]: I am not sure is that how it resolves the quotes issue, as there is a single quote `can't` within those outer quotes.

**Pull request merged**